packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableTokenImage.tsx

```diff
@@ -0,0 +1,66 @@
+import { useMemo } from 'react'
+import EthereumLogoRoundLight from '@/images/EthereumLogoRoundLight.svg'
+import Image from 'next/image'
+
+import { useTokenLists } from '../../hooks/useTokenLists'
+import { MergedTransaction } from '../../state/app/state'
+import { orbitChains } from '../../util/orbitChainsList'
+import { AssetType } from '../../hooks/arbTokenBridge.types'
+
+export const TransactionsTableTokenImage = ({
+  tx
+}: {
+  tx: MergedTransaction
+}) => {
+  // we need to take token image from mainnet by symbol, some token images don't exists on other networks
+  const tokenLists = useTokenLists(tx.sourceChainId)
+
+  const tokenAddressToLogoSrcMap = useMemo(() => {
```

Comment:
nit, but we can extract this into a function, like `createTokenLogoMapFromTokenLists`

Reply:
updated